test/spec/util/router.spec.js

```diff
@@ -0,0 +1,19 @@
+(function () {
+    'use strict';
+
+    define(
+        [
+            'chai',
+            'util/router'
+        ],
+        function (chai, router) {
+            var expect = chai.expect;
+
+            describe('The `router` module', function () {
```

Comment:
This one needs tests but I haven't quite figured out how yet, without adding extra code to the app just to support tests (e.g. abstraction layer between the app and `page` library, which could then be mocked out for the tests).